### ISSUE TITLE
version 0.12 - additional support for macOS intel?

### ISSUE THREAD

**Issue by suschizu** [FPGAwars/icestudio#751]
Will there also be a version for Intel Macs?
the download 
https://github.com/FPGAwars/icestudio/releases/download/v0.12/icestudio-0.12-osxarm64.dmg  
contains an ARM-only version.
thanks!

**Comment by placessweden** [comment]
I wonder the same. To give up the Intel-port is ridiculous because, in the case of Apple Silicon, it will run Intel-binaries just fine using the Rosetta 2 framework.

**Comment by cavearr** [comment]
Hi! first of all, we don't had beed feeback from users  with intel osx and we can't do tests for it for the new versions.

About the Rosetta. framework, it has a very bad performance for nwjs engine.

If you could give me  feedback for osx intel  tests i'll working in the dmg and use you as testers.

Are you ready for the tests?? ;)  Give me a couple of days and prepare the dmg bundle for intel osx, i'm telling you in this thread ok?

**Comment by placessweden** [comment]
I'm very happy if I can help you testing! Today I have messed around with the source with grunt-cli and npm (which I've never used in my life :) which give me two dependency errors, grunt-wget and grunt-appdmg

**Comment by cavearr** [comment]
If you need help tell me.

To start icestudio from cli from the sources if you need help, tell me.

You only need to execute:

npm install --legacy-peer-deps

and when finish it:

npm start

**Comment by placessweden** [comment]
Thankyou! Works like a charm on my Intel-Mac. However, there are complaints that grunt-appdmg is missing.
<img width="897" alt="Icestudio-0 12-intel" src="https://github.com/FPGAwars/icestudio/assets/111527798/16dbd967-d8ca-430c-83e0-8eb7641ec5ce">

**Comment by cavearr** [comment]
Yes, don't worry, this is one of the problems to generate the intel dmg. I'm working on updating all the packages because we use some "abandoned" packages, and I'm working on a lot of breaking changes and these packages are going away soon.

Thank you and in the next few days I will tell you to try the intel dmg package. I need your comments, so I can stabilize it. Many other users will be very grateful to you!

**Comment by placessweden** [comment]
Thanks! 👍

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
